main.go

```diff
@@ -622,13 +622,16 @@ func transformCompile(args []string) ([]string, error) {
 		return nil, err
 	}
 
-	if curPkg.ImportPath == "runtime" && opts.Tiny {
-		// When using -tiny, we call stripRuntime below.
-		// We don't want -literals and -debugdir to apply, though.
+	// in the runtime don't obfuscate literals (x escapes to heap, not allowed in runtime)
```

Comment:
It took me a few reads to understand what you mean here. How about:

```
// We can't obfuscate literals in the runtime and its dependencies,
// because obfuscated literals sometimes escape to heap,
// and that's not allowed in the runtime itself.
```

I also wonder why the runtime doesn't allow it. I guess because it manages the heap and GC, so it must only use the stack?